### PR TITLE
Minor (docs): Update how-to-use-the-playground.md document

### PR DESCRIPTION
**Pull request by unknowntpo**
### What changes were proposed in this pull request?

Revert helm-chart related descriptions in `docs/how-to-use-the-playground.md`.

### Why are the changes needed?

Make documentation match `README.md` of `apache/gravitino-playground` .

### Does this PR introduce _any_ user-facing change?

Yes, documentation is affacted.

### How was this patch tested?

Manually.

### CODE REVIEW

**Comment by unknowntpo** [comment]
@xunliu Please have a look.

File: docs/how-to-use-the-playground.md [hunk 1]
```diff
@@ -50,82 +49,22 @@ git clone git@github.com:apache/gravitino-playground.git
...
 ```
 
-#### Docker
-
-##### Start
+#### Start
 
 ```
 ./playground.sh docker start
 ```
```
Comment (xunliu) [inline docs/how-to-use-the-playground.md:56]: Please update here.

Comment (unknowntpo) [reply docs/how-to-use-the-playground.md]: fixed at a63cd7cc3575a8e7de5a6ffc733354797e419e26.

**Review by jerryshao — APPROVED** [review]
LGTM, @xunliu do you want to take another look?

**Review by xunliu — APPROVED** [review]
@unknowntpo Thank you for your contributions.
LGTM

**Pull request merged**